### PR TITLE
[Mellanox] add test cases for dynamic minimum fan speed and psu fan speed policy

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
### Description of PR

Summary:
Add test cases for dynamic minimum fan speed policy and PSU fan speed policy.

### Type of change

- [  ] Bug fix
- [  ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### How did you do it?

1. Add a test case to test dynamic minimum fan speed policy. This policy mocks random air flow direction and random thermal zone temperature. It checks that the minimum fan speed value is equal to expect value.

2. Add a test case to test PSU fan speed policy. This policy mocks max cooling level, minimum cooling level and compare the PSU fan speed on different situation.

#### How did you verify/test it?

Manually run the two test cases.

#### Any platform specific information?

Mellanox platform.

#### Supported testbed topology if it's a new test case?

All existing topologies.

### CODE REVIEW

**Comment by Junchao-Mellanox** [comment]
Depends on https://github.com/Azure/sonic-buildimage/pull/4403

File: tests/platform/mellanox/minimum_table.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+MINIMUM_TABLE= {
+    'ACS-MSN2700': {
+        "p2c_trust":   {"-127:40":13, "41:120":15},
+        "p2c_untrust": {"-127:25":13, "26:30":14 , "31:35":15, "36:120":16},
+        "c2p_trust":   {"-127:20":13, "21:25":14 , "26:30":15, "31:120":16},
+        "c2p_untrust": {"-127:20":13, "21:25":14 , "26:30":15, "31:120":16},
+        "unk_trust":   {"-127:20":13, "21:25":14 , "26:30":15, "31:120":16},
+        "unk_untrust": {"-127:20":13, "21:25":14 , "26:30":15, "31:120":16}
+    },
+    'LS-SN2700': {
```
Comment (wangxin) [inline tests/platform/mellanox/minimum_table.py:10]: Where does this LS-SN2700 come from?

**Comment by jleveque** [comment]
@Junchao-Mellanox: Looks good to me. Can you please answer the open question?

File: tests/platform/mellanox/minimum_table.py (hunk 1 above)
Comment (Junchao-Mellanox) [reply tests/platform/mellanox/minimum_table.py:10]: It's one SKU of 2700, I just copy the SKU name from existing code.
